packages/cli/src/index.ts

```diff
@@ -19,7 +18,7 @@ export function defineConfig(config: Config): ConfigInit {
         return tokenPath;
       }
       try {
-        return new URL(`file://${require.resolve(tokenPath)}`);
+        return pathToFileURL(join(process.cwd(), tokenPath));
```

Comment:
this error'd when the token didnt start with `./`, like in the case of token-example.json in the cli package folder.

Reply:
hm we actually do need the `require.resolve` here to deal with resolving `package.json` imports correctly; some paths are rewritten. but I can take a look at it later.